gatsby/content/projects/clients/chooj.mdx

```diff
@@ -0,0 +1,31 @@
+---
+layout: projectimage
+title: chooj
+categories:
+# valid categories are client, server, sdk (client sdk), bot, bridge, iot and other
+ - client
+description: Describe your project here
+slug: chooj
+author: Farooq Karimi Zadeh
+home: https://github.com/farooqkz/chooj
+language: Javascript
+license: GPL-3.0-or-later
+# pick your license from https://spdx.org/licenses/
+repo: https://github.com/farooqkz/chooj # your source code repository
+room: "#chooj:mozilla.org" # nb the quotes are needed as rooms start with #
+thumbnail: /docs/projects/images/chooj-screenie.png
+screenshot: /docs/projects/images/chooj-screenie.png
+maturity: Early Alpha # Options are: Released, Stable, Late Beta, Beta, Early Beta, Late Alpha, Alpha, Early Alpha, or Not actively maintained
```

Comment:
maturity is Early Alpha, but the repository project says

> The project is inactive because I must spend my time on projects which make me real money.

Do you confirm the project is currently maintained :) ?

Reply:
Oh you are right. Didn't notice there was `Not actively maintanied` available as well. Gotta change it to that.